site/owid.entry.ts

```diff
@@ -82,6 +85,43 @@ if (BUGSNAG_API_KEY) {
     }
 }
 
+if (SENTRY_DSN) {
+    try {
+        const analyticsConsent = getPreferenceValue(PreferenceType.Analytics)
+
+        let sentryOpts: Sentry.BrowserOptions
+        if (analyticsConsent && !isInIFrame()) {
+            // only collect session replays from: users that have consented to analytics
+            // AND where page isn't embedded in an iframe
+            sentryOpts = {
+                integrations: [
+                    Sentry.replayIntegration({
+                        maskAllText: false,
+                        maskAllInputs: false,
+                        blockAllMedia: false,
+                        mask: [".sentry-mask"],
+                    }),
+                ],
+                replaysSessionSampleRate: ENV === "development" ? 1 : 0.01,
+                replaysOnErrorSampleRate: 0,
```

Comment:
Capturing replays on error seems even more useful than non-errors. Is there a good reason to disable it?

Reply:
It's more so the opposite for me, as I don't want our budget of session replays to get inadvertently consumed by a few pages that raise a lot of the same error over and over (e.g. we end up with 1k monthly session replays on `/search` out of our 5k budget just b/c of errors).

Obviously, keeping `replaysOnErrorSampleRate` low would help avoid this problem, but it was easier for now to just set this to 0 to get a clear baseline of how well the 5k monthly replays are distributed across our pages (without having to worry about how much errors skew the distribution of replays).